packages/amplify-category-geo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amplify-category-geo",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
are you manually bumping this version? If so, you don't need to, you can just let lerna manage the versions when publishing